src/Form.vue

```diff
@@ -86,7 +97,8 @@ export default {
       question.isValid = true;
       question.validationMessage = "";
     },
-    getComponentByQuestionType(questionType) {
+    getComponentByQuestionType(question) {
+      const questionType= question.guiOptions && question.guiOptions.type ? question.guiOptions.type : (question.guiType ? question.guiType : question.type);
       const foundPlugin = this.plugins.find(plugin => {
         return plugin.questionType === questionType;
       });
```

Comment:
If plugin is not found for provided gui type, look for it again using type (see [unrelated branch](https://github.com/SAP/inquirer-gui/blob/hrtt/src/Form.vue#L91-L105) for example)